pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -75,21 +89,29 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     interface IDeveloperIdProvider
     {
+        AuthenticationExperienceKind GetAuthenticationExperienceKind(); 
+
         String GetName();
 
         IIterable<IDeveloperId> GetLoggedInDeveloperIds();
 
+        // This API is deprecated and should not be used. The API will be removed once all callers have been moved over.
```

Comment:
Is it a good idea to remove an API?  Would that be considered a breaking chage?

Reply:
Resolved in offline thread